src/mongocxx/collection.cpp

```diff
@@ -123,25 +124,26 @@ collection::operator bool() const noexcept {
     return static_cast<bool>(_impl);
 }
 
-stdx::string_view collection::name() const {
-    return stdx::string_view{libmongoc::collection_get_name(_get_impl().collection_t)};
+std::string collection::name() const {
```

Comment:
Why did this need to change? It seems that the underlying mongoc collection object keeps its string around, so it should be fine to return a view of that?